tests/conftest.py

```diff
@@ -131,7 +133,7 @@ def tf_version(request):
     return request.param
 
 
-@pytest.fixture(scope='module', params=['1.12', '1.12.0'])
+@pytest.fixture(scope='module', params=['1.11', '1.11.0', '1.12', '1.12.0'])
```

Comment:
this seems to go against what we've decided for everything else

Reply:
removed